docs/gallery/interactive/pyfar_arithmetics.ipynb

```diff
@@ -0,0 +1,237 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pyfar as pf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Arithmetic operations using two or more audio objects\n",
+    "\n",
+    "Consider the two signals below for illustrating the use of arithmetic operations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "signal_1 = pf.Signal([1, 0, .5, 0], 44100)\n",
+    "signal_2 = pf.Signal([2, -2, 0, 0], 44100)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "They can be added using `pf.add` in the time and frequency domain"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result = pf.add((signal_1, signal_2), domain='time')\n",
+    "print(f'Time domain addition:\\nresult.time = {result.time}\\n')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result = pf.add((signal_1, signal_2), domain='freq')\n",
+    "print(f'Frequency domain addition:\\nresult.time = {result.time}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this case it does not matter if the operation is performed in the time or frequency domain, because the FFT and the addition are linear and time invariant operations. But this is not true for all arithmetic operations as will become clear later. All arithmetic operations are performed in the frequency domain by default and pyfar overloads the standard arithmetic operators. The same operation can thus be done with"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result = signal_1 + signal_2\n",
+    "print(f'Frequency domain addition:\\nresult.time = {result.time}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "All frequency domain operations on `pyfar.Signal` obejcts work on `signal.freq_raw`, that is, the signal spectrum without normalization of the Discrete Fourier Transform. If using the same signals for a multiplication, it becomes clear that the domain makes a difference in this case"
```

Comment:
```suggestion
    "If using the same signals for a multiplication, it becomes clear that the domain makes a difference"
```

Reply:
I'd suggest to move "All frequency domain operations on `pyfar.Signal` objects work on `signal.freq_raw`, that is, the signal spectrum without normalization of the Discrete Fourier Transform." as first sentence of the DFT normalization section.

Reply:
Yes, good idea